org.eclipse.jface.text/src/org/eclipse/jface/internal/text/html/BrowserInformationControl.java

```diff
@@ -169,6 +169,9 @@ public static boolean isAvailable(Composite parent) {
 	 */
 	private final String fSymbolicFontName;
 
+	private int fDisposeTimeout= 0;
```

Comment:
There are 3 different constructors, do you prefer this line to be duplicated in all 3? Or maybe there is something else we can do that I'm missing?

Reply:
`private int fDisposeTimeout= 0;` and `private int fDisposeTimeout;` does literally same **except** in debugger one will jump here if we do explicit init.

Reply:
Yes, it generates additional code that you will notice if you debug, and of course such code is pointless bloat if you initialize to the same value it will have when not initialized.